.github/workflows/manage-neon-database.yml

```diff
@@ -0,0 +1,51 @@
+name: Manage Neon Database
+
+# This workflow triggers on every push to the "main or preprod" branch
+on:
+  push:
+    branches:
+      - preprod
+      - main
+
+jobs:
+  create-db:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: "20"
+
+      - name: Install Neon CLI
+        run: npm install -g neonctl
+
+      - name: Install PostgreSQL Client
+        run: sudo apt-get install -y postgresql-client
+
+      - name: Create Neon Database
+        env:
+          NEON_API_KEY: ${{ secrets.NEON_API_KEY }}
+        run: |
+          # Create a new branch and database on Neon
+          neonctl create-branch --project-id ${{ secrets.NEON_PROJECT_ID }} --branch-name development
```

Comment:
Why do we need to create a branch ? 
And maybe you can dynamically name the branch because you're always creating the same branch

Reply:
You're absolutely right; creating a branch every time wasn't necessary for our workflow. I've removed the branch creation step entirely since we can use the existing branches, which keeps things simpler and avoids redundancy. Thank you for pointing that out!